devs/docs/es-backports.rst

```diff
@@ -141,26 +141,26 @@ should be crossed out as well.
 - [ ] 43ace5f80d7 Emit deprecation warnings when boosts are defined in mappings (#62623)
 - [ ] 9f5e95505bf Also abort ongoing file restores when snapshot restore is aborted (#62441) (#62607)
 - [ ] 06d5d360f92 Tidy up fillInStackTrace implementations (#62555)
-- [ ] 9bb7ce0b229 [7.x] Allocate new indices on "hot" or "content" tier depending on data stream inclusion (#62338) (#62557)
```

Comment:
We don't have hot tier nodes, we also skipped 
`Allocate newly created indices on data_hot tier nodes (#61342) (#61650)`